packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * @file Transaction Resolver - handles cross-chain transaction confirmations
+ *
+ * This resolver is responsible for tracking pending transactions and providing
+ * promises/vows that resolve when transactions are confirmed.
+ * This is an orchestration component that can be used independently of portfolio logic.
+ */
+
+import { makeTracer, mustMatch } from '@agoric/internal';
+import type { CaipChainId, AccountId } from '@agoric/orchestration';
+import type { Zone } from '@agoric/zone';
+import { type VowTools, type VowKit, VowShape } from '@agoric/vow';
+import type { ZCF, ZCFSeat, Invitation } from '@agoric/zoe';
+import { M } from '@endo/patterns';
+import type {
+  CCTPTransactionKey,
+  CCTPSettlementOfferArgs,
+} from './types.js';
+import { ResolverOfferArgsShapes } from './types.js';
+import { q, X } from '@endo/errors';
+
+type CCTPTransactionEntry = {
+  destinationAddress: AccountId;
+  amountValue: bigint;
+  vowKit: VowKit<void>;
+};
+
+const trace = makeTracer('Resolver');
+
+const ClientFacetI = M.interface('CCTPResolverClient', {
+  registerCCTPTransaction: M.call(M.string(), M.nat()).returns(VowShape),
+});
+
+const ServiceFacetI = M.interface('CCTPResolverService', {
+  settleCCTPTransaction: M.call(
+    M.string(),
+    M.string(),
+    M.bigint(),
+    M.string(),
+  ).returns(),
+});
+
+const InvitationMakersFacetI = M.interface('ResolverInvitationMakers', {
+  SettleCCTPTransaction: M.callWhen().returns(M.remotable()),
+});
+
+/**
+ * Make CCTP Resolver Kit.
+ *
+ * This creates a unified resolver with all CCTP functionality including
+ * transaction registration, settlement, and invitation makers.
+ *
+ * @param resolverZone - Durable storage zone
+ * @param zcf - Zoe Contract Facet for creating invitations
+ * @param vowTools - Vow tools for creating promises
+ * @returns Resolver kit with client, service, and invitationMakers facets
+ */
+export const makeResolverKit = (
+  resolverZone: Zone,
+  zcf: ZCF,
+  vowTools: VowTools,
+) => {
+  const cctpTransactionRegistry = resolverZone.mapStore<
+    CCTPTransactionKey,
+    CCTPTransactionEntry
+  >('cctpTransactionRegistry');
+
+  return resolverZone.exoClassKit(
```

Comment:
`exoClassKit` _returns_ a maker function, which implies that function returning that is not itself such a maker function. See also [A Taxonomy of Exo-making Functions: Make/Define vs Prepare](https://github.com/endojs/endo/blob/master/packages/exo/docs/exo-taxonomy.md#makedefine-vs-prepare).
```suggestion
export const prepareResolverKit = (
  resolverZone: Zone,
  zcf: ZCF,
  vowTools: VowTools,
) => {
  const cctpTransactionRegistry = resolverZone.mapStore<
    CCTPTransactionKey,
    CCTPTransactionEntry
  >('cctpTransactionRegistry');

  return resolverZone.exoClassKit(
```

Reply:
oops! my mistake!